docker-compose.yml

```diff
@@ -4,8 +4,16 @@ services:
   web:
     container_name: varaamo-frontend
     build:
+      target: development
       context: .
       dockerfile: Dockerfile
+      args:
+        API_URL: https://dev-respa.tampere.fi/v1
+        RESPA_ADMIN_URL: https://dev-respa.tampere.fi/ra/
+        LOGIN_CALLBACK_URL: https://dev-varaamo.tampere.fi/login/tampere/return
+        AUTH_LOGOUT_URL: https://dev-auth.tampere.fi/logout/
+        PORT: 3000
+
     volumes:
       - '.:/usr/src/app'
       - '/usr/src/app/node_modules'
```

Comment:
Old Dockerfile copied to that path now we should use the /app path for these volumes.